ntt/parameters/alt_bn128.h

```diff
@@ -3,109 +3,209 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #if defined(__GNUC__) || defined(__clang__)
-# define FR_T(vec_t, ...) fr_t{(vec_t){__VA_ARGS__}}
+#define FR_T(vec_t, ...)                                                       \
+  fr_t {                                                                       \
+    (vec_t) { __VA_ARGS__ }                                                    \
+  }
 #else
-# define FR_T(vec_t, ...) fr_t{vec_t{__VA_ARGS__}}
+#define FR_T(vec_t, ...)                                                       \
+  fr_t {                                                                       \
+    vec_t { __VA_ARGS__ }                                                      \
+  }
```

Comment:
nit: you could revert those if it's pure reformatting.

Since it's a fork of sppark, we want to minimize our footprints.